app/templates/partials/widgets/multiple_choice_widget.html

```diff
@@ -35,12 +40,20 @@
     <div class="field__item js-focusable-box">
       <input {{input|xmlattr}}>
       <label {{label|xmlattr}}>
-        {{option.label.text}}
+          {% if not exclusive or (exclusive and not loop.last) %}
+            {{option.label.text}}
+          {% endif %}
         {% if answer.options[loop.index0].description %}
           <br />
           <span class="label__description label__inner pluto">{{answer.options[loop.index0].description}}</span>
         {% endif %}
+          {% if exclusive and loop.last %}
```

Comment:
Slightly different context I think. The logic is around wether something should be shown on the page or not. A filter would be used to change what is show not decide if it should be shown or not. We did consider having a separate widget for mutually exclusive but this would have led to a lot of duplication so a bit of logic here seemed the best option.

Reply:
Good point to raise. I agree that this is probably a fair balance in this case though.

If the template were to become more complex we could look to refactor later but as it is I don't think it's holding too much logic nor does it take responsibility for any logic that it shouldn't be.